pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/AggregationFunctionType.java

```diff
@@ -65,6 +65,7 @@ public enum AggregationFunctionType {
   STDDEVSAMP("stdDevSamp"),
   SKEWNESS("skewness"),
   KURTOSIS("kurtosis"),
+  FOURTHMOMENT("fourthmoment"),
```

Comment:
Not sure I follow this. Why did we need to introduce this as an individual aggregate function type ? 

My understanding is that `FOURTHMOMENT` by itself is not exposed as an aggregation function that can be used in SQL queries and instead `skewness` and `kurtosis` are wrapped under `fourthmoment`

Reply:
calcite will literally rewrite the query into `SKEWNESSREDUCE(FOURTHMOMENT(col))`, which means the engine needs to know how to execute this - so I register it as `FOURTHMOMENT` directly. I'm not sure of a way to avoid exposing this altogether, open to suggestions though.

Reply:
with the throw exception change you added above i think this would be meaningful exception for end-user for now. we can look into how to hide non-user-facing functions later?

Reply:
So `SELECT fourthMoment(..) FROM FOO` will throw exception informing the user that it is an illegal / invalid aggregation function ? 

In that case, I am fine with leaving here and later on figuring out how to hide it.

Reply:
cc @agavra

Reply:
yes, that's correct 👍